### DISCUSSION TITLE
Conscious Community Capital: A Real Estate Fund for Conscious Community

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#982] [Ideas]
This is a thread for collecting work we did from 2019-2022 around creating a real estate fund for supporting conscious coliving spaces. This work was a joint effort in collaboration with Ian Sneath and Mark Bogues.

Whilst we are no longer pursuing this exact approach, we plan to build on it in various going forward. As a first step we want to collect past material and back-publish it for reference.

### Conscious Community Capital Overview (this version Dec 2021)

An outline developed over 2020-2021.

[Conscious Community Capital - A Real Estate Fund for Conscious Community - Summary.pdf](https://github.com/life-itself/community/files/13770825/Conscious.Community.Capital.-.A.Real.Estate.Fund.for.Conscious.Community.-.Summary.pdf)

**Comment by martinprogrammer** [comment]
@rufuspollock 
I love this initiative!  Very interested in the idea.  Is this similar to what Cabin are doing (except that they are doing it through a DAO)?
Have you got any more papers on this?

**Reply by rufuspollock** [reply]
> I love this initiative! Very interested in the idea. ... Have you got any more papers on this?

@martinprogrammer yes, i have a lot more material. I will post it up as i can.

> Is this similar to what Cabin are doing (except that they are doing it through a DAO)?

I don't know Cabin so well - i just saw the posts on the general chat a while ago (writing up what they did would be useful btw - i started doing a bit of excerpting myself which i posted). There definitely seemed some similarities ... 

> Have you got any more papers on this?

PS: If you would like to help work through it and post it let me know and we can have a quick chat and i can show you the google drive folder and walk you through what is worth posting ...

PPS: thank-you for the enthusiasm it is a real encouragement 😉

**Reply by martinprogrammer** [reply]
> PS: If you would like to help work through it and post it let me know and we can have a quick chat and i can show you the google drive folder and walk you through what is worth posting ...

Sure, just let me know what's needed ;-)
